src/controllers/Controller.js

```diff
@@ -0,0 +1,112 @@
+import { Console } from "@woowacourse/mission-utils";
+
+import Opponent from "../models/Opponent.js";
+import User from "../models/User.js";
+
+import GAME_START_VIEW from "../views/GameStartView.js";
```

Comment:
import할 때 확장자를 생략하면 다른 환경의 시스템에서 파일 확장자를 변경해도 동일한 코드를 작성할 수 있어 호환성을 확보할 수 있습니다!

```suggestion
import GAME_START_VIEW from "../views/GameStartView";
```

Reply:
저도 처음에 그렇게 했는데 IDE 상에서 eslint 문법에 어긋난다 그래서 고쳤습니다 ㅠ 그런데 airbnb 컨벤션에도 말씀 주신 것처럼 확장자를 빼는 것을 추천하는 것 같아서 다시 한 번 확인해봐야겠네요! 감사합니다:)